tools/bug-report/pkg/bugreport/bugreport.go

```diff
@@ -198,6 +212,43 @@ func runBugReportCommand(_ *cobra.Command, logOpts *log.Options) error {
 	return nil
 }
 
+func isDistroless(ctx context.Context, client kube.CLIClient, clientset *kubernetes.Clientset, brConfig *config.BugReportConfig) (bool, error) {
+	// Get istiod deployment
```

Comment:
Added logic to get the effective IOP from multiple available IOPs using `istiod` deployment. I am not sure if it is the best possible approach to determine the effectiev IOP for the installation. Suggestions/Feedback would be really helpful here. Thanks.